### ISSUE TITLE
Command not found: impl

### ISSUE THREAD

**Issue by lzrf0cuz** [ricardoerikson/vscode-go-impl-methods#4]
Issue: `/bin/sh: impl not found`.
This seems to be the [issue.](https://stackoverflow.com/questions/38542947/bin-sh-1-node-not-found-with-child-process-exec)

**Comment by ricardoerikson** [comment]
Yeah, you need to install `impl` first. There is a requirements section in the description with some instructions on how to do that. I'll make some improvements on this.

```
go get -u github.com/josharian/impl
```

Let me know if it works with this.

**Comment by lzrf0cuz** [comment]
I have it installed, and use it all the time from the terminal. I was so happy to find this extension since it offers a convenient dropdown. The child_prosess for some reason doesn't find the binary. 
Please view the which output.

> which impl
/Users/redacted/go/bin/impl

**Comment by happyxhw** [comment]
```
cp $HOME/go/bin/impl /bin/
```
this works, but abnormal
